R/subset_cnstates.R

```diff
@@ -2,7 +2,7 @@
 #'
 #' @description Get the available CN states in the incoming data frame.
 #'
-#' @details For sub-setting copy number information based on segments available in cn data
+#' @details INTERNAL FUNCTION called by [GAMBLR::fancy_multisample_ideo], for sub-setting copy number information based on segments available in cn data
```

Comment:
Same here, just want to make sure this is not called by devtools as dead link

Reply:
The `fancy_multisample_ideo` function was removed from GAMBLR and is not present in any child repo. I deleted the `@details` field of the documentation of this function.

Reply:
This function is available in GAMBLR.viz. However, it's called `fancy_multisamp_ideogram`.

Reply:
Thanks for this information, @mattssca.

But as @Kdreval commented somewhere else in this PR:

> "... we do not necessarily need to refer to a specific package in description"

I assume that there is no need to add the following `@details` tag to this function documentation:

```
#' @details INTERNAL FUNCTION called by [GAMBLR.viz::fancy_multisamp_ideogram], for sub-setting copy number information based on segments available in cn data
```

Reply:
Right, it was more generic before these edits
```
#' @details For sub-setting copy number information based on segments available in cn data
```
and I think it is fine to keep as is

Reply:
It's not like `#' @details For sub-setting copy number information based on segments available in cn data` on master anymore. That was before Laura's merged PR.